acl/tests/test.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+set -euo pipefail
+#/ Usage: test.sh <pkg_ident>
```

Comment:
You'll want this right under the shebang starting on line 2 👍

Reply:
Amended.